magenta/models/sketch_rnn/sketch_rnn_train.py

```diff
@@ -20,13 +20,14 @@
 import json
 import os
 import time
-import urllib
+import urllib.request
 import zipfile
 
 import numpy as np
 import requests
 import six
-from six.moves import cStringIO as StringIO
+#from six.moves import cStringIO as StringIO
+from io import BytesIO
```

Comment:
please use six.BytesIO

Reply:
On documentation, it says six.BytesIO is a "fake file object for binary data." I'm not sure if this is a package. Correct me if I'm wrong, of course.

Reply:
catherine, does your change as-is work with python2 and python3?

Reply:
based on the documentation, it looks like @cghawthorne was correct:
`This is a fake file object for binary data. In Python 2, it’s an alias for StringIO.StringIO, but in Python 3, it’s an alias for io.BytesIO.`

Reply:
The reason I made these changes is because the previous version didn't work for python3. After these changes, it worked with Python3.

Reply:
however, I didn't test for python2

Reply:
I'm going to try again with six.bytesio